pkg/db/src/types.ts

```diff
@@ -102,12 +104,7 @@ export interface NoteProto<A extends Record<string, any> = {}> {
 	// Note specific methods
 	/** Set name updates the `displayName` of the note. */
 	setName: (name: string, ctx: debug.DebugCtx) => Promise<NoteInterface<A>>;
-	/**
-	 * Add volumes accepts an array of volume stock transactions and adds them to the note.
-	 * If any transactions (for a given isbn/warehouse) already exist, the quantity gets aggregated.
-	 */
-	addVolumes: (...params: VolumeTransactionTuple | VolumeTransactionTuple[]) => Promise<NoteInterface<A>>;
-	/** Explicitly update an existing transaction row. */
+	addVolumes: (...params: VolumeStockOptionalWarehouseId[]) => Promise<NoteInterface<A>>;
 	updateTransaction: (transaction: VolumeStock) => Promise<NoteInterface<A>>;
```

Comment:
Can you update the comments here (the previous comments were fine, just update them with the updates you've applied to the function)